test/pini_test.cpp

```diff
@@ -5,8 +5,10 @@
 int main() {
 	pn::pini pin;
 
-	std::string_view raw_input{"a=5\nb=10\n23=c"};
+	std::string_view raw_input{"= 5\n#b=10\n c  = 23"};
 	// std::filesystem::path filename{"/test/test.ini"};
 	if (!pin.load_text(raw_input)) { return 1; }
-	if (pin.get_int32("a") != 5) { return 1; }
+	std::cout << pin.get_int32("a");
+	std::cout << pin.get_int32("b");
+	std::cout << pin.get_int32("c");
```

Comment:
I'd put some more failure cases instead of just printing stuff: you won't see these logs via ctest / through CI etc.